index.js

```diff
@@ -1 +1,40 @@
 // Code your solution in this file!
+function distanceFromHqInBlocks(destination){
+    if(destination > 42){
+        return destination - 42
+    }
+    else{
+        return 42 - destination
+    }
+}
+
+function distanceFromHqInFeet(destination){
+    if(destination > 42){
+        return (destination - 42) * 264
+    }
+    else{
+        return (42 - destination) * 264
+    }
```

Comment:
Instead of duplicating the logic here, you can levage the the `distanceFromHqInBlocks` function:
```suggestion
  distanceFromHqInBlocks(destination) * 264
```